gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -1595,8 +1588,7 @@ private String listStorageObjectsAndPrefixesPage(
 
     // Set delimiter if supplied.
     if (delimiter != null) {
-      listObject.setDelimiter(delimiter);
-      listObject.setIncludeTrailingDelimiter(includeTrailingDelimiter);
+      listObject.setDelimiter(delimiter).setIncludeTrailingDelimiter(true);
```

Comment:
Is this change(removing `includeTrailingDelimiter`) necessary ? 

Seems unrelated change, could we separate this into a different PR if it is intentional ?

Reply:
Reverted.